### PR TITLE
hover on donut shows percentage

### LINKED ISSUE #1728: v3.0.0 Campaign analytics missing data available in previous versions
**Version:**
 - listmonk: v3.0.0
 - OS: Ubuntu (Docker)

**Description of the bug and steps to reproduce:**
Campaign analytics missing percentage that was available in previous versions. Please close this ticket if this is intentional.

**Screenshots:**
Mouse hover
image

Entire page
image

### PR DESCRIPTION
**Pull request by Bowrna**
closes: #1728 
related: #1728

### CODE REVIEW

**Comment by MaximilianKohler** [comment]
Is it difficult to make it similar to the previous design where you didn't have to hover to view the numbers? 

Eg: https://github.com/knadh/listmonk/issues/1728#issuecomment-1987235485

File: frontend/src/components/Chart.vue [hunk 1]
```diff
@@ -146,6 +161,7 @@ export default {
     }
 
     const conf = { ...def, data: this.$props.data };
+    console.log("Inside Chart.vue:",conf);
```
Comment (knadh) [inline frontend/src/components/Chart.vue:164]: This should be removed.

File: frontend/src/views/CampaignAnalytics.vue [hunk 2]
```diff
@@ -269,8 +266,8 @@ export default Vue.extend({
 
     getData(typ, camps) {
       this.charts[typ].loading = true;
-
       // Call the HTTP API.
+
```
Comment (knadh) [inline frontend/src/views/CampaignAnalytics.vue:270]: This superfluous line break can be removed.

**Comment by Bowrna** [comment]
@knadh Resolved the issues.

**Comment by Bowrna** [comment]
Let me know if I have to fix anything in this PR.

**Comment by knadh** [comment]
Thanks @Bowrna. Will test it this week and merge.

**Comment by knadh** [comment]
Merged. Thanks!

PS: Made a slight tweak and removed the custom truncated label as the tooltip shows the label in bold by default anyway.

image

**Comment by Bowrna** [comment]
> Merged. Thanks!
> 
> PS: Made a slight tweak and removed the custom truncated label as the tooltip shows the label in bold by default anyway.
> 
> image

My bad! it didn't strike me that label is already visible and I should have removed it rather than truncating 🫤

**Comment by MaximilianKohler** [comment]
This is still broken, but I'm not sure how to fix it myself: https://github.com/knadh/listmonk/issues/1728#issuecomment-4380857084

**Pull request merged**